File: 02_automatedSimplifiedExtensionLayer/00_matlab/functions/get_missing_meta_data.m
function [missing_meta_data] = get_missing_meta_data(path_own_scenario)

        file_name=fullfile(path_own_scenario,'meta_data_flows.xlsx');
        
        [~,~, missing_meta_data]=xlsread(file_name);
        
         missing_meta_data(find(cellfun(@(C)...
             any(isnan(C(:))), missing_meta_data)))={[]}; 
   
end

File: 02_automatedSimplifiedExtensionLayer/utils/get_missing_meta_data.py
import pandas as pd
import numpy as np
import os

def get_missing_meta_data(path_own_scenario):
    # Construct the file path
    file_name = os.path.join(path_own_scenario, 'meta_data_flows.xlsx')
    
    # Read the Excel file into a pandas DataFrame
    missing_meta_data = pd.read_excel(file_name, header=None).values.tolist()
    
    # Replace NaN values with empty lists (equivalent to MATLAB's [])
    missing_meta_data = [[None if pd.isna(cell) else cell for cell in row] for row in missing_meta_data]
    
    return missing_meta_data
